applications/GeoMechanicsApplication/tests/cpp_tests/test_soil_density.cpp

```diff
@@ -0,0 +1,42 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_utilities/transport_equation_utilities.hpp"
+#include "includes/checks.h"
+#include "testing/testing.h"
```

Comment:
I'm not sure if we need the containers/model.h, could we try to remove it?
```suggestion
#include "custom_utilities/transport_equation_utilities.hpp"
#include "includes/checks.h"
#include "testing/testing.h"
```

Reply:
You are right, yes we can and I made it now.